### PR TITLE
Add test logger.

### PR DESCRIPTION
**Pull request by dzeckelev**
It can work in two modes: mute or verbose.
Resolves BV-1194.

### CODE REVIEW

File: util/log/test_logger.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		WriterConfig: NewWriterConfig(),
+	}
+}
+
+// NewTestLogger creates a new test logger.
+func NewTestLogger(conf *TestConfig) (Logger, error) {
+	if conf.Verbose {
+		return NewStderrLogger(conf.WriterConfig)
+	}
+	return NewWriterLogger(conf.WriterConfig, ioutil.Discard)
```
[the remark this answers is not in the document]
Comment (dzeckelev) [reply util/log/test_logger.go]: Why do we need `log.NewMultiLogger()`? We write only to stderr.

[the remark this answers is not in the document]
Comment (ababo) [reply util/log/test_logger.go]: Not to write anywhere.

File: util/log/test_logger.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package log
+
+import "io/ioutil"
+
+// TestConfig is a test logger configuration.
+type TestConfig struct {
+	*WriterConfig
+	Verbose bool
```
[the remark this answers is not in the document]
Comment (dzeckelev) [reply util/log/test_logger.go]: This is a good idea.

File: util/log/test_logger.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (ababo) [reply util/log/test_logger.go]: (i.e. to call it with no arguments)

File: util/log/test_logger.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (dzeckelev) [reply util/log/test_logger.go]: @ababo updated

File: util/log/test_logger.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (furkhat) [reply util/log/test_logger.go]: I tried it and it gives nil pointer error. @dzeckelev if you could fix that error then a having `log.NewMiltiLogger()` call with no arguments would be much simpler.

[the remark this answers is not in the document]
Comment (dzeckelev) [reply util/log/test_logger.go]: The behavior was due to conflicts. This Is fixed.

**Pull request merged**